Clinical-Genomics/scout
# Refactor: Capitalize Search Menu

This PR adds a functionality, capitalise search menu for increased readability.

**How to test**:
1. Install branch `refactor/capitalize_menu`
2. Go to Case page and click search. See images for comparison.

**Expected outcome**:
The functionality should be working

Before:

<img width="921" alt="Screenshot 2021-10-14 at 11 25 34" src="https://user-images.githubusercontent.com/1150065/137291712-929b1709-358d-49b3-968f-765afb29be3b.png">

After:
<img width="932" alt="Screenshot 2021-10-14 at 11 35 46" src="https://user-images.githubusercontent.com/1150065/137291873-76fa5a9f-6473-4002-947d-8896d996a886.png">

**Review:**
